airbyte-webapp/src/locales/en.json

```diff
@@ -522,6 +522,7 @@
   "connector.connectorsInDevelopment.docLink": "See our <lnk>documentation</lnk> for more details.",
   "connector.setupGuide": "Setup Guide",
   "connector.setupGuide.notFound": "No Setup Guide found for this connector.",
+  "connector.optionsHeader": "Options",
```

Comment:
I think all of the changes to files in `airbyte-webapp` should not be included in this PR